### PR TITLE
Reuse generated code in E2E

### PR DESCRIPTION
**Pull request by janisz**
Since we generate API client in jenkins-plugin and upload `jar` to repository there is no need to regenerate client in e2e tests. We can reuse previously generated client. This will save build time, reduce number of dependencies and reuse plugin code giving us more coverage.

### CODE REVIEW

File: functionaltest-jenkins-plugin/build.gradle [hunk 1]
```diff
@@ -23,54 +23,23 @@ repositories {
     mavenCentral()
     maven {
         url 'https://repo.jenkins-ci.org/releases'
+        metadataSources {
+            artifact()
+        }
```
Comment (janisz) [inline functionaltest-jenkins-plugin/build.gradle:28]: Since uploaded `jar` has no `pom` we need to change metadata sources.

**Comment by janisz** [comment]
Current dependencies on/for this PR:
* master
  * **PR #236** <a href="https://app.graphite.dev/github/pr/stackrox/jenkins-plugin/236" target="_blank">Graphite</a> 
    * **PR #235** <a href="https://app.graphite.dev/github/pr/stackrox/jenkins-plugin/235" target="_blank">Graphite</a> 
      * **PR #237** <a href="https://app.graphite.dev/github/pr/stackrox/jenkins-plugin/237" target="_blank">Graphite</a>  👈
        * **PR #240** <a href="https://app.graphite.dev/github/pr/stackrox/jenkins-plugin/240" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/jenkins-plugin/237?utm_source=stack-comment).

**Review by dhaus67 — APPROVED** [review]

**Comment by janisz** [comment]
@janisz queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/stackrox/jenkins-plugin/237).

**Comment by janisz** [comment]
[Graphite](https://app.graphite.dev/github/pr/stackrox/jenkins-plugin/237) failed to merge this pull request for an unknown reason. Try your merge again, or report a bug if you see this consistently.

**Comment by janisz** [comment]
@janisz queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/stackrox/jenkins-plugin/237).

**Comment by janisz** [comment]
[Graphite](https://app.graphite.dev/github/pr/stackrox/jenkins-plugin/237) rebased this pull request as part of a merge.

**Comment by janisz** [comment]
[Graphite](https://app.graphite.dev/github/pr/stackrox/jenkins-plugin/237) failed to merge this pull request for an unknown reason. Try your merge again, or report a bug if you see this consistently.

**Comment by janisz** [comment]
@janisz merged this pull request with [Graphite](https://app.graphite.dev/github/pr/stackrox/jenkins-plugin/237).

**Pull request merged**